setup.py

```diff
@@ -0,0 +1,65 @@
+"""
```

Comment:
This is the main file that allows `pip install`. It has been verified that it can be installed from another project.